apollo-router/src/plugins/external.rs

```diff
@@ -72,6 +236,8 @@ struct RouterStage {
 struct Stages {
     /// The router stage
     router: Option<RouterStage>,
```

Comment:
This was here before, but in general avoid `Option` in config and prefer implementing `Default`. This goes for both `Conf`, `Stages` and `RouterConf`.

Reply:
fixed!